integration-tests/test_ftp.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import unicode_literals
+
+import logging
+import unittest
+
+from smart_open import open
+
+class FTPTest(unittest.TestCase):
```

Comment:
We use pytest, so you don't need a test class.  Just define the test_ functions at module level.